src/commands/retention-policies/assign.js

```diff
@@ -11,14 +11,22 @@ class RetentionPoliciesAssignCommand extends BoxCommand {
 
 		let assignType = flags['assign-to-type'];
 		let assignID = flags['assign-to-id'];
+		let options = {};
+
+		if (flags.hasOwnProperty('filter-fields')) {
```

Comment:
Can you also extend `RetentionPoliciesAssignCommand.examples` description with new added flags?